pkg/storegateway/gateway.go

```diff
@@ -39,6 +39,11 @@ const (
 	// ringAutoForgetUnhealthyPeriods is how many consecutive timeout periods an unhealthy instance
 	// in the ring will be automatically removed.
 	ringAutoForgetUnhealthyPeriods = 10
+
+	// ringNumTokensDefault is the number of tokens registered in the ring by each store-gateway
```

Comment:
This comment is no longer accurate because this change _does_ expose the config to the user :smile:

Reply:
ack @56quarters let me update the comment